packages/framework/attributor/src/mixinAttributor.ts

```diff
@@ -147,17 +145,38 @@ export const mixinAttributor = (Base: typeof ContainerRuntime = ContainerRuntime
 			)) as ContainerRuntimeWithAttributor;
 			runtime.runtimeAttributor = runtimeAttributor as RuntimeAttributor;
 
+			const logger = ChildLogger.create(runtime.logger, "Attributor");
+			const mc = loggerToMonitoringContext(context.taggedLogger);
+
+			const shouldTrackAttribution = mc.config.getBoolean(enableOnNewFileKey) ?? false;
+			if (shouldTrackAttribution) {
+				(context.options.attribution ??= {}).track = true;
+				// Add telemetry to track if the storage of attribution data was enabled in configuration or not
+				logger.sendTelemetryEvent({ eventName: "load", attributionEnabledInConfig: true });
```

Comment:
i'd just roll this property into the other event.

Reply:
I moved this telemetry event after the load of runtime, then I changed the eventName as well since "load" may not be appropriate here anymore.

Reply:
i meant it'd probably be fine to just use a single event: just add a property to the perf event below and remove this one entirely.

The config setting is actually also available elsewhere in our tables (Mark did a brownbag on this on Tuesday), so it's mostly for convenience here anyway.

Reply:
Ahh got it. I think we can directly use something like `event.end({..., attributionEnabledInConfig: shouldTrackAttribution,});` Since the `shouldTrackAttribution` is extracted from the `mc.config`